examples/misc/ftface_props.py

```diff
@@ -3,9 +3,9 @@
 Font properties
 ===============
 
-This example lists the attributes of an `.FT2Font` object, which describe
-global font properties.  For individual character metrics, use the `.Glyph`
-object, as returned by `.load_char`.
+This example lists the attributes of an ``FT2Font`` object, which describe
```

Comment:
Changing references to literal is good for old changelogs when the referenced code is removed. In other cases, there are often better solutions. I'm away from my dev machine, so cannot easily check but I suspect that we have made these private. If so, we shouldn't mention them in the docs anymore at all.

Reply:
Not sure if they are private or if there is limited documentation for the classes that originates from C.

Reply:
Ok, this is coming from C, and we never had documentation for that. Literal is good enough here.